### ISSUE TITLE
Model test accuracy is very low while train accuracy is 99%

### ISSUE THREAD

**Issue by pathak-ashutosh** [m3rc3n4ry/plant-seeds-classification#1]
The model is giving a training accuracy of 99% but the test set accuracy is very low (~10%).
Is it overfitting? Or, is the shuffling not done correctly?
